### PR TITLE
Add Delete and Update 

### PR DESCRIPTION
**Pull request by erimow**

### CODE REVIEW

File: src/components/AddAssignment.js [hunk 1]
```diff
@@ -1,11 +1,69 @@
 import React, { useState } from 'react';
-
+import {SERVER_URL} from '../constants';
+import {Link} from 'react-router-dom';
 
 function AddAssignment(props) {
```
Comment (louison2) [inline src/components/AddAssignment.js:6]: State for displaying messages to the user

File: src/components/AddAssignment.js [hunk 2]
```diff
@@ -1,11 +1,69 @@
 import React, { useState } from 'react';
-
+import {SERVER_URL} from '../constants';
+import {Link} from 'react-router-dom';
 
 function AddAssignment(props) { 
 
+  const [message, setMessage] = useState('');
+  const [assign, setAssign] = useState([]);
```
Comment (ghost) [inline src/components/AddAssignment.js:8]: State to store list of assignments

File: src/components/AddAssignment.js [hunk 3]
```diff
@@ -1,11 +1,69 @@
...
+  const [factors, setFactors] = useState({assName:"", assId:0, classId:0, assDueDate:"", courseName:""});
+
+  const saveAssignment = ( ) => {
```
Comment (ghost) [inline src/components/AddAssignment.js:11]: Function to save the assignment to the server

File: src/components/AddAssignment.js [hunk 4]
```diff
@@ -1,11 +1,69 @@
 import React, { useState } from 'react';
-
+import {SERVER_URL} from '../constants';
+import {Link} from 'react-router-dom';
 
 function AddAssignment(props) { 
 
+  const [message, setMessage] = useState('');
```
Comment (ghost) [inline src/components/AddAssignment.js:7]: State for displaying messages to the user

File: src/components/AddAssignment.js [hunk 5]
```diff
@@ -1,11 +1,69 @@
...
+          setMessage("Save error. "+res.status);
+          console.error('Save assignment error =' + res.status);
+    }})
+      .catch(err => {
+          setMessage("Exception. "+err);
+          console.error('Save assignment exception =' + err);
+      });
+ };        
+
+ const handleChange = (e) => {
```
Comment (ghost) [inline src/components/AddAssignment.js:41]: Handle changes to input fields and update state accordingly

**Review by ghost — APPROVED** [review]
Approved

**Pull request merged**